wordpress-mobile/gutenberg-mobile
# Dynamically search for plugin-proposal-async-generator-functions

`Gutenberg:` https://github.com/WordPress/gutenberg/pull/25588

Fixes an error related to missing module `@babel/plugin-proposal-async-generator-functions`
<img width=300 src="https://user-images.githubusercontent.com/3384451/94035015-a135d880-fd90-11ea-8387-59d788156399.png"/>

### To test:
- `npm install`
- `npm start`
- Check that the editor loads
- Stop Metro
- `npm run core start`
- Check that the editor loads

## PR submission checklist:
